src/openassetio-python/package/pyproject.toml

```diff
@@ -119,7 +119,7 @@ target-version = ["py39"]
 
```

Comment:
[question] I was in two minds whether to move these files under `openassetio-python/package` or just under `openassetio-python`.  

In particular, any configuration related to running tests or linters applies to `openassetio-python/tests/package` too.

But having this under `package` seems neater, since the primary responsibility of `pyproject.toml`/`setup.py` is related to  packaging the `package`.

:question:

Reply:
Hmm, i'm not really in a strong mind about it. My intuition would be to have them directly under `openassetio-python`, rather than under package, but I don't have strong arguments. What I've got is: 

- There's a `CMakeLists.txt` at that level already, meaning it dosen't "look" like an expectable python project when you go into the python directory, without these files
- Not totally sure how well the idea that because `setup.py`/`pyproject.toml` are involved in packaging, that means they must go in `package`. `package` is the noun here, whilst `setup.py` is the verb, one could almost make the strained argument that the packager should be outside of the package.
- I can imagine a possibility`setup.py` wanting to reach into adjacent folders at some point in the future, for whatever reason, and having it top level makes that a tad more natural

As usual when there's no strong arguments either way, I yield to the author, happy either way.

Reply:
Good arguments, moved one level up in 22b434d9.  The `package` directory looks a bit lonely and sparse now, but I do kinda like that `setup.py`/`pyproject.toml` is above the `module`, `package` and `tests`, which are all related.

I guess @foundrytom will be back before this is merged anyway, so we can get a tie-break opinion.  Easy enough to drop the fixup if the previous location is preferable.

Reply:
We chatted together about this, and decided on keeping it under `openassetio-python`, as it helps convey that the `package` subdir is not self-contained.